source/common/runtime/runtime_features.cc

```diff
@@ -25,6 +25,9 @@ namespace Runtime {
 constexpr const char* runtime_features[] = {
     // Enabled
     "envoy.reloadable_features.test_feature_true",
+    "envoy.reloadable_features.strict_header_validation",
+    "envoy.reloadable_features.http2_protocol_options.max_inbound_priority_frames_per_stream",
+    "envoy.reloadable_features.http2_protocol_options.max_outbound_frames",
```

Comment:
These aren't actual reloadable features, but integer overrides, so I don't these belong here. I think I commented on this during the CVE code review, but we probably need a different name for this type of thing or some other way to tag it so the scripts ignore it? WDYT?

Reply:
Oh yuck.  Is the plan to deprecate these as we do the booleans?
I definitely agree I want to be consistent about types here - I think given our plans to mark things true by default, I'd be inclined to tweak tooling (probably the check/fix_format scripts) that if you add a flag it has to exist in runtime features or be specifically excluded.
that'll both help enforce true-by-default from here on, as well as boolean types

Reply:
IMO these should stay forever to give operators the ability to emergency override the defaults. FWIW this is why I was suggesting a different name for these. We can potentially still rename them as long as we release note / announce it? I doubt very many people are setting them.

Reply:
I'd just remove them. Why would operators need an emergency override for those settings, now that the HTTP/2 fixes turned out to be working correctly? cc @yanavlasov

Reply:
Because the fixes are heuristic based and might not work for all applications. Granted, one could argue people should test their apps ahead of time and tweak the config, but it feels like leaving the emergency overrides in place for a while can't hurt. With that said if folks prefer removing them I won't object.

Reply:
There is plenty of other settings in Envoy that might break the application, for which we don't have emergency overrides. I agree that it was a good call to add then during the initial rollout, when we weren't 100% sure that the fixes are not going to break anything, but the defaults are very conservative and we didn't hear any complains in the 3 months since the release, so I don't see a point in leaving those _emergency_ overrides in the codebase, to be honest.

Reply:
Fair enough, SGTM.

Reply:
@alyssawilk do you want to just do the deletion in this PR? WDYT?

Reply:
If we're giving behavioral changes 6 months, don't we want to wait one more release for the removal?
I think we land this as-is, and then remove those in ~3 months.

Reply:
:+1: